manager/controlapi/service.go

```diff
@@ -141,6 +141,18 @@ func validateContainerSpec(container *api.ContainerSpec) error {
 	return nil
 }
 
+func validatePluginSpec(plugin *api.PluginSpec) error {
+	if plugin.Image == "" {
+		return grpc.Errorf(codes.InvalidArgument, "PluginSpec: image reference must be provided")
+	}
+
+	if _, err := reference.ParseNormalizedNamed(plugin.Image); err != nil {
```

Comment:
@aaronlehmann I was able to install a plugin by just providing the digest (but it still adds the default tag `latest`) -- unlike images though, where the image may not have a tag.